src/makeCalculator.js

```diff
@@ -4,7 +4,57 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      const number = this;
+
+      number.result += num;
+
+      return number;
```

Comment:
```suggestion
      this.result += num;

      return this;
```
just use `this` directly